### ISSUE TITLE
JavaScript strings are not matched when multiline.

### ISSUE THREAD

**Issue by gonsfx** [PrismJS/prism#146]
Multiline Example:
```javascript
var str = 'This is a \
multiline string';
```
Doesn't match.

My suggestion:
```javascript
/("|')(?:.|\\\1|\\\s)*?\1/g
```

**Comment by loganfranken** [comment]
Looks like this works! :confetti_ball:
https://jsfiddle.net/9wL6ywx4/

I think we can close this one!

**Comment by Golmote** [comment]
You're right, this was fixed with commit 3aeecb5a83b181d4218ebd6555bcd1ddb5003c3b.
